includes/Core/Authentication/Setup.php

```diff
@@ -104,6 +105,22 @@ public function register() {
 		add_action( 'admin_action_' . Google_Proxy::ACTION_EXCHANGE_SITE_CODE, array( $this, 'handle_action_exchange_site_code' ) );
 	}
 
+	/**
+	 * Composes the oAuth proxy get help link.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The get help link.
+	 */
+	private function get_oauth_proxy_failed_help_link() {
+		return sprintf(
+			/* translators: 1: Support link URL. 2: Get help string. */
+			__( '<a href="%1$s" target="_blank">%2$s</a>', 'google-site-kit' ),
+			esc_url( $this->proxy_support_link_url . '/?error_id=request_to_auth_proxy_failed' ),
```

Comment:
Please use `add_query_arg()` to add query parameters instead of manually concatenating pieces of a URL.

Not only is that a best practice in WordPress, it is also more error proof: Right now, the code here only works as long as we know or can assume that `$this->proxy_support_link_url` does not end in a trailing slash. With `add_query_arg`, we don't have that problem.

Reply:
Duly noted and very valid concern. Thank you for noticing and pointing it out @felixarntz!